src/marias3.c

```diff
@@ -534,6 +535,19 @@ uint8_t ms3_set_option(ms3_st *ms3, ms3_set_option_t option, void *value)
       break;
     }
 
+    case MS3_OPT_PORT_NUMBER:
+    {
+      int port_number;
+
+      if (!value)
+      {
+        return MS3_ERR_PARAMETER;
+      }
+      memcpy(&port_number, (void*)value, sizeof(int));
+
+      ms3->port = port_number;
```

Comment:
Is there some benefit to doing a memcpy into a temporary variable over casting the pointer to `int*`?

Reply:
Yes. I am not sure if the pointer is aligned.

Reply:
alignment, I suppose